sampledkd/training/entrypoint_utils.py

```
@@ -0,0 +1,369 @@
+            model_name,
+            device_map={"": one},
+            quantization_config=q8,
+            low_cpu_mem_usage=True,
+            trust_remote_code=True,
+        )
+        teacher.eval()
+        for p in teacher.parameters():
+            p.requires_grad_(False)
+        print("[teacher] Loaded 8-bit.", flush=True)
+        return teacher, tok, torch.device(f"cuda:{one}")
+    except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)